src/List/ItemList.js

```diff
@@ -39,7 +62,20 @@ const Items = props => {
       ) : (
         <ul>
           {dbItems.map((item, index) => {
-            return <li key={index}>{item.name}</li>;
+            return (
+              <li key={index}>
+                <label>
+                  <input
+                    type="checkbox"
+                    name="isPurchased"
+                    checked={is24Hours(item)}
+                    onChange={e => handleChange(e)}
+                    value={item.name}
+                  />
+                  {item.name}
+                </label>
+              </li>
+            );
```

Comment:
As a user, it may not be clear to me what the checkbox is actually for, as there is no label/instructions that indicate to check the box once purchased.

Reply:
Oh, really good point! We'll be adding delete buttons this week so it would be a good to clarify the checkboxes while we're doing that. Thanks!